sdk/src/wallet/operations/output_consolidation.rs

```diff
@@ -80,6 +122,11 @@ where
             let protocol_parameters = self.client().get_protocol_parameters().await?;
             let unlock_conditions = basic_output.unlock_conditions();
 
+            // Implicit account creation outputs shouldn't be consolidated.
+            if basic_output.address().is_implicit_account_creation() {
```

Comment:
So we do consolidate outputs with other addresses and that's fine ?

Reply:
That's the current behavior in stardust, not sure if anyone has a problem with it or not